labs/cckit.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: CCKit
+parent: Labs
+---
+
+# Lab Name
+
+[CCKit](https://github.com/hyperledger-labs/cckit)
+
+# Short Description
+
+Programming framework for building Hyperledger Fabric Golang on-chain (chaincode) and off-chain applications&
+
+# Scope of Lab
+
+The scope of this lab is to deliver the framework for building Hyperledger Fabric Golang on-chain (chaincode) and
+off-chain applications:
+
+* Routing chaincode function invocation with middleware options
+* Chaincode state mapping and serializing
+* Chaincode testing
+* Defining chaincode interface with gRPC / Protobuf notation
+* Generating chaincode gRPC/REST access layer
+
+# Initial Committers
+
+- https://github.com/vitiko
+- https://github.com/dimaxgl
+- https://github.com/bogatyr285
+- https://github.com/inotnako
+- https://github.com/TopHatCroat
+- https://github.com/criro1
+- https://github.com/uraldav
+- https://github.com/czar0
+- https://github.com/darkleaf
+- https://github.com/larsvliet
+- https://github.com/kukgini
+- https://github.com/kolayuk
+- https://github.com/stepanshurkov
+
+# Sponsor
+
+- https://github.com/C0rWin - HLF maintainer
+
+# Pre-existing repository
+
+- https://github.com/s7techlab/cckit
```

Comment:
It looks like some of the commits in this repository do not contain DCO signoff. Hyperledger requires that all contributed code has a DCO sign off. Looking at these repositories, it appears that this is not the case. There are two options that we normally consider for fixing this:

1. Amend existing commits in the repository to include DCO sign off
1. Manually transfer the code from the existing repo to the newly created repo within the hyperledger-labs

Secondly, it appears that this code is currently MIT licensed. According to [Hyperledger's charter](https://hyperledger.org/about/charter):
> Members agree that all new inbound code contributions to HLP shall be made under the Apache License, Version 2.0 (available at http://www.apache.org/licenses/LICENSE-2.0). All contributions shall be accompanied by a Developer Certificate of Origin sign-off (http://developercertificate.org/) that is submitted through a Governing Board and LF-approved contribution process. Such contribution process will include steps to also bind non-Member Contributors and, if not self-employed, their employer, to the licenses expressly granted in the Apache License, Version 2.0 with respect to such contribution.

Reply:
Hi, @tkuhrt ! Thank you for comments.  
The license issue is [fixed](https://github.com/s7techlab/cckit/blob/master/LICENSE).  
Transferring the code - I prefer the second option (manually transfer the code from the existing repo to the newly created repo within the hyperledger-labs)